test/dotnet.Tests/CommandTests/ToolInstallCommandTests.cs

```diff
@@ -57,7 +57,8 @@ public ToolInstallCommandTests()
                     filePermissionSetter: new NoOpFilePermissionSetter());
             _environmentPathInstructionMock =
                 new EnvironmentPathInstructionMock(_reporter, _pathToPlaceShim);
-            _createToolPackageStoreAndInstaller = (_) => (_toolPackageStore, CreateToolPackageInstaller());
+            _createToolPackageStoreAndInstaller =
+                (location, forwardArguement) => (_toolPackageStore, CreateToolPackageInstaller());
```

Comment:
Nit: spelling of `forwardArguement`.

Reply:
Repeat this comment for each use below.

Reply:
no, but i have to change to (_, __) it looks too funny. What is the convention here?

Reply:
Yeah, let's not use that (I sure wish C# treated `_` as special like other languages).  See original comment above for edited suggestion.

Reply:
fixed, replaced all to restoreArguments